apps/web/src/app/(dashboard)/documents/[id]/document-page-view.tsx

```diff
@@ -162,13 +167,17 @@ export const DocumentPageView = async ({ params, team }: DocumentPageViewProps)
           </CardContent>
         </Card>
 
-        {document.status === DocumentStatus.PENDING && (
+        {document.status === DocumentStatus.COMPLETED && (
           <DocumentReadOnlyFields
             fields={completedFields}
             documentMeta={document.documentMeta || undefined}
           />
         )}
 
+        {document.status === DocumentStatus.PENDING && (
+          <DocumentPendingFields fields={pendingFields} />
+        )}
```

Comment:
Can't we just use `DocumentReadOnlyFields`?

Reply:
This appears to be identical

Just reuse `DocumentReadOnlyFields` and update it to be more generic